lib/build/lb.js

```diff
@@ -0,0 +1,90 @@
+const express = require('express');
+const _ = require('lodash');
+const lib = require('../utils');
+const serverUtils = require('../server-utils');
+const crypto = require('crypto');
+class LbBuilder {
+
+    constructor(labShareRC, config) {
+        this.labShareRC = labShareRC;
+        this.config = this.getDefaultConfig(config);
+    }
+    /**
+     * Builds the apis using loopback
+     */
+    async build() {
+        this.app = express();
+        const lbServices = new lib.Lb(this.labShareRC, this.config);
+        const lbApis = lbServices.loadAPIs();
+        const { morgan, security, bodyParser } = this.config.services;
+        serverUtils.initializeSecurity({ security, expressApp: this.app });
+        serverUtils.initializeBodyParser({ bodyParser: bodyParser, expressApp: this.app });
+        if (morgan.enable) {
+            serverUtils.initializeMorgan({ morgan: morgan, expressApp: this.app });
+        }
+
+        serverUtils.initializeCookieParser({ expressApp: this.app });
+
+        // Set up express-session middleware
+        serverUtils.initializeSessions(
+            {
+                sessionOptions: security.sessionOptions,
+                logger: console,
+                expressApp: this.app
+            });
+
+        for (const api of lbApis) {
+            this.app.use(api.config.basePath || api.name, api.app.requestHandler);
+            await api.app.boot();
+        }
+        this.app.use(require('compression')());
+        this.app.use(require('cors')());
```

Comment:
This sets CORS "*", we probably don't want to force that on the APIs.

Reply:
what about set it on configuration ?

Reply:
Sure, we can configure it using the Lb4 "rest" object.